pocket_coffea/parameters/lepton_scale_factors.yaml

```diff
@@ -2,28 +2,59 @@
 lepton_scale_factors:
   ####################################################################################################
   electron_sf:
-    reco:
-      pt_gt_20: RecoAbove20
-      pt_lt_20: RecoBelow20
 
-    id: wp80iso
+    id: 
```

Comment:
For documentation reasons, it would be nice to add here a comment to explicit that the map here is linking the SFname for Iso and the label that should be used in the object preselections.  

Just to refresh my memory: why don't we use the same label for the object preselection to avoid this map?

Reply:
The label for "id", that is used in the object preselection is the name of the corresponding field in NanoAOD (see e.g. [here](https://cms-nanoaod-integration.web.cern.ch/autoDoc/NanoAODv12/2022/2023/doc_EGamma_Run2022F-22Sep2023-v1.html#Electron) . This id requirement is also checked directly without further mapping [here](https://github.com/PocketCoffea/PocketCoffea/blob/main/pocket_coffea/lib/leptons.py#L21). Therefore, if one were to use the "sf labels" in the object preselection, one would have to introduce a mapping to the corresponding NanoAOD field, which is i think less intuitive.